src/sas/qtgui/Plotting/SlicerParameters.py

```diff
@@ -128,6 +134,62 @@ def updatePlotList(self):
         self.active_plots = self.parent.getActivePlots()
         self.setPlotsList()
 
+    def getCurrentSlicerDict(self):
+        """
+        Returns a dictionary of currently shown slicers
+        {slicer_name:checkbox_status}
+        """
+        current_slicers = {}
+        if self.lstSlicers.count() != 0:
+            for row in range(self.lstSlicers.count()):
+                item = self.lstSlicers.item(row)
+                isChecked = item.checkState() != QtCore.Qt.Unchecked
+                slicer = item.text()
+                current_slicers[slicer] = isChecked
+        return current_slicers
+
+    def setSlicersList(self):
+        """
+        Create and initially populate the list of slicers
+        """
+        current_slicers = self.getCurrentSlicerDict()
+        self.lstSlicers.clear()
+
+        # Fill out list of slicers
+        for item in self.parent.slicers:
+            if str(item) in current_slicers.keys():
+                # redo the list
+                checked = QtCore.Qt.Checked if current_slicers[item] else QtCore.Qt.Unchecked
+            else:
+                # create a new list
+                checked = QtCore.Qt.Checked if (self.parent.data[0].name == item) else QtCore.Qt.Unchecked
+
+            rbItem = QtWidgets.QListWidgetItem(str(item))
+            # Change to radio button behavior - only one selected at a time
+            rbItem.setFlags(QtCore.Qt.ItemIsUserCheckable | QtCore.Qt.ItemIsEnabled)
+            rbItem.setCheckState(checked)
+            self.lstSlicers.addItem(rbItem)
+
+            # Store the slicer's model
+            if item in self.parent.slicers:
+                slicer_obj = self.parent.slicers[item]
+                if hasattr(slicer_obj, '_model'):
+                    self.slicer_models[str(item)] = slicer_obj._model
+
+    def getCheckedSlicer(self):
+        """
+        Returns the currently checked slicer
+        """
+        checked_slicer = None
+        if self.lstSlicers.count() != 0:
+            for row in range(self.lstSlicers.count()):
+                item = self.lstSlicers.item(row)
+                isChecked = item.checkState() != QtCore.Qt.Unchecked
+                slicer = item
+                if isChecked:
+                    checked_slicer = slicer
+        return checked_slicer
```

Comment:
Since there's only one checked item, you can simplify this

```python
        for row in range(self.lstSlicers.count()):
            item = self.lstSlicers.item(row)
            if item.checkState() != QtCore.Qt.Unchecked:
                return item
        return None
```